File: src/main/java/org/zenbot/timetableupdater/domain/BusStop.java
package org.zenbot.timetableupdater.domain;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Data
@Document
public class BusStop {

    @Id
    private String id;
    private String busStopName;
    private Schedule workDaySchedule;
    private Schedule saturdaySchedule;
    private Schedule sundaySchedule;

}

File: src/main/kotlin/org/zenbot/timetableupdater/domain/BusStop.kt
package org.zenbot.timetableupdater.domain

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document
import java.lang.reflect.Constructor

@Document
data class BusStop (
        @Id var id: String? = null,
        var busStopName: String = "",
        var workDaySchedule: Schedule = Schedule(),
        var saturdaySchedule: Schedule = Schedule(),
        var sundaySchedule: Schedule = Schedule()
)